aws-production-2/main.tf

```diff
@@ -103,9 +136,16 @@ module "aws_asg_org" {
   worker_asg_scale_in_threshold = 16
   worker_asg_scale_out_qty = 2
   worker_asg_scale_out_threshold = 8
+  worker_cache_bucket = "${var.worker_org_cache_bucket}"
```

Comment:
These cache buckets are used inside the jobs themselves, e.g. `cache: bundler`.  The bucket, key, and secret all used to be injected, and with this change only the bucket is injected while the IAM bits are generated in terraform land.